### PR TITLE
Feature: Uniform request naming

### PR DESCRIPTION
**Pull request by MaxAndreassenRPM**
Base on our discussion yesterday, I've updated the naming of the base requests to be more uniform and reflective of the actual API resources.

image

### CODE REVIEW

File: Runtime/AvatarCreator/WebRequests/AuthorizedRequest.cs [hunk 1]
```diff
@@ -23,15 +21,8 @@ public async Task<T> SendRequest<T>(RequestData requestData, CancellationToken c
...
-                {
-                    await AuthManager.RefreshToken();
-                }
-                catch (Exception)
-                {
-                    throw;
-                }
-                
+                await AuthManager.RefreshToken();
+
```
Comment (rYuuk) [inline Runtime/AvatarCreator/WebRequests/AuthorizedRequest.cs:25]: I would suggest not removing this, if I remember correctly that this was added for when even RefreshToken request fails and then without it the error is not caught and propagated above. But somehow rider always suggest to remove it.

**Review by rYuuk — APPROVED** [review]
🟢 Thanks this makes api more consistent. Please also have a look at a small concern.

File: Runtime/AvatarCreator/WebRequests/AuthorizedRequest.cs (hunk 1 above)
Comment (MaxAndreassenRPM) [reply Runtime/AvatarCreator/WebRequests/AuthorizedRequest.cs:25]: Oh, interesting, it seemed to be redundant but if it's a weird edge case then I'll add it back 👍

Comment (MaxAndreassenRPM) [reply Runtime/AvatarCreator/WebRequests/AuthorizedRequest.cs]: Added back, and also added a comment on it too so others don't make the same assumption in the future that I did.

**Pull request merged**